explore/src/main/scala/explore/targeteditor/AsterismEditor.scala

```diff
@@ -61,7 +62,12 @@ object AsterismEditor {
     selectedTarget: View[Option[Target.Id]]
   )(implicit ctx:   AppContextIO): Callback = {
     val (targetId, createTarget) = oTargetId.fold(
-      (newId, ((tid: Target.Id) => TargetQueries.createSiderealTarget[IO](tid, target)))
+      (newId,
```

Comment:
This was the only thing left in `TargetQueries.scala`, so I just moved it over here and deleted that file.